public/dist/js/controllers/AnnouncementController.js

```diff
@@ -1,9 +1,47 @@
 app.controller('AnnouncementController', ['$scope', '$http', function($scope, $http) {
+	let getAnnouncements = function(){
```

Comment:
This should be `const` or `var`. We actually shouldn't be using ES6 standards in the front-end as it would break on non-modern browsers such as IE.